src/pds/registrysweepers/ancestry/main.py

```diff
@@ -88,140 +76,75 @@ def run(
             updates,
             index_name=resolve_multitenant_index_name(client, "registry"),
         )
-        log.info("Generating updates from deferred records...")
-        deferred_updates = generate_deferred_updates(client, deferred_records_file.name, registry_mock_query_f)
 
-        log.info("Writing deferred updates to database...")
-        write_updated_docs(
-            client,
-            deferred_updates,
-            index_name=resolve_multitenant_index_name(client, "registry"),
-        )
     else:
         # consume generator to dump bulk updates to sink
         for _ in updates:
             pass
 
-    log.info("Checking indexes for orphaned documents")
-    index_names = [resolve_multitenant_index_name(client, index_label) for index_label in ["registry", "registry-refs"]]
-    for index_name in index_names:
-        if log.isEnabledFor(logging.DEBUG):
-            orphaned_docs = get_orphaned_documents(client, registry_mock_query_f, index_name)
-            orphaned_doc_ids = [doc.get("_id") for doc in orphaned_docs]
-            orphaned_doc_ids_str = str(orphaned_doc_ids)
-            orphaned_doc_count = len(orphaned_doc_ids)
-        else:
-            orphaned_doc_ids_str = "<run with debug logging enabled to view list of orphaned lidvids>"
-
-            # Currently, mocks are only implemented for iterating over document collections, not accessing the
-            # enclosing query response metadata.  This is a shortcoming which should be addressed, but in the meantime
-            # this bandaid will allow functional tests to complete when a client is not provided, i.e. during functional
-            # testing.
-            # TODO: refactor mock framework to provide access to arbitrary queries, not just the hits themselves
-            def orphan_counter_mock(_, __):
-                return -1
-
-            orphan_counter_f = get_orphaned_documents_count if client is not None else orphan_counter_mock
-            orphaned_doc_count = orphan_counter_f(client, index_name)
-
-        if orphaned_doc_count > 0:
-            log.warning(
-                f'Detected {orphaned_doc_count} orphaned documents in index "{index_name} - please inform developers": {orphaned_doc_ids_str}'
-            )
+    # TODO: reimplement orphan checking for ancestry sweeper - edunn 20251112
```

Comment:
Would that be ok to implement that in a later release or do we need to have the orphan management done as we compute also the ancestry ?